stubs/gevent/@tests/stubtest_allowlist_win32.txt

```diff
@@ -1,4 +1,6 @@
-# TODO: missing from stub
+# Error: is not present in stub
+# =============================
+# these get exported but don't actually work on win32 so we ignore them
 gevent.signal.__all__
 
 # Error: is not present in stub
```

Comment:
I can't comment/suggest all the way down to line 8. But this comment section is not duplicated.
`gevent.signal.__all__` can go with `gevent.signal.getsignal` and `gevent.signal.signal`. They're all omitted together for the same reasons.